Installation/CHANGES.md

```diff
@@ -6,6 +6,12 @@ Release 5.0
 
 Release date: September 2019
 
+### 2D and 3D Linear Geometry Kernel
+ - Added `ComputeApproximateAngle_3` in the 2D/3D Kernel concept to compute
+   the approximate dihedral angle between 2 vectors. Corresponding functors
+   in the model (`Compute_approximate_angle_3`) and free function (`approximate_angle`)
+   are also added.
+
```

Comment:
I do not understand this changes of the changelog. Does this PR really add `ComputeApproximateAngle_3`?!

Reply:
This comes from here:
https://cgal.geometryfactory.com/CGAL/Members/wiki/Features/Small_Features/approximate_angle

Reply:
So it came from https://github.com/CGAL/cgal/pull/3468 and not from this PR. Why is it in the changes, then?

Reply:
PMP and Kernel were inverted initially in the PR so in the merge conflict resolution I kept that order, which is more natural than having kernel changes at the bottom. The removal corresponding to this addition is actually 3 lines below. 

It doesn't work so well due to other changes being added to kernel with kernel at a different stop, so now it's a bit messy, I'll clean it.